CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 0.6.0
+- Add support for typed screen arguments on deep links
```

Comment:
Maybe worth adding the list of support types here?

Reply:
Done!